bundle/manifests/rhods-operator.clusterserviceversion.yaml

```
@@ -98,16 +99,9 @@ metadata:
     capabilities: Full Lifecycle
     categories: AI/Machine Learning, Big Data
     certified: "False"
-    containerImage: quay.io/opendatahub/opendatahub-operator:v2.0.0
-    createdAt: "2024-10-15T10:03:01Z"
-    features.operators.openshift.io/disconnected: "true"
-    features.operators.openshift.io/fips-compliant: "false"
-    features.operators.openshift.io/proxy-aware: "false"
-    features.operators.openshift.io/tls-profiles: "false"
-    features.operators.openshift.io/token-auth-aws: "false"
-    features.operators.openshift.io/token-auth-azure: "false"
-    features.operators.openshift.io/token-auth-gcp: "false"
-    olm.skipRange: '>=1.0.0 <2.0.0'
+    containerImage: REPLACE_IMAGE:latest
+    createdAt: "2025-02-11T13:28:45Z"
+    olm.skipRange: ">=2.13.0 <2.13.1"
```

i think it is for 2.13.2
```suggestion
    olm.skipRange: ">=2.13.0 <2.13.2"
```